modules/ocf_mirrors/manifests/centos.pp

```diff
@@ -8,10 +8,19 @@
     recurse => true,
   }
 
-  ocf_mirrors::monitoring { 'centos':
+  ocf_mirrors::monitoring {
+    default:
       type          => 'unix_timestamp',
       upstream_host => 'mirror.centos.org',
-      ts_path       => 'TIME',
+      ts_path       => 'TIME';
+
+    'centos':
+    ;
```

Comment:
I feel like this is confusing as it is, I'd just prefer a few things to be duplicated here instead of how it is now. Then at least it would be easier to read/maintain.